.github/workflows/wheel_tests_and_release.yml

```diff
@@ -5,6 +5,7 @@ on:
       - v*
     branches:
       - maintenance/**
+      - "!main"
```

Comment:
Just curious about the context of this change? The way I understand [GitHub's docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore) `maintenance/**` would be enough to not build on `main`..?

Reply:
That was my intuition as well, but not what we've been seeing in practice 🤷‍♂️